src/components/Cell.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+interface CellContent {
+    classes: {
+        cell: string
+    },
+    letter: null
+}
+
+export const Cell: React.FC<CellContent> = ({ letter, classes }) => {
```

Comment:
It's more organized to receive a props argument with a CellContent (CellProps) signature; i.e: 
`export const Cell: React.FC<CellContent> = (props: CellContent) => {`

Reply:
And then get the fields you need in a `const { bla, ble, bli } = props;`

Reply:
Question, is it a good practice create an interface inside the component? Or is it better to create it in a separate folder?